schemas/context/media-timed-asset-view-details.schema.json

```diff
@@ -26,6 +26,16 @@
           "type": "string",
           "description": "Name of the content player."
         },
+        "xdm:videoSegment": {
+          "title": "Video segment",
+          "type": "string",
+          "description": "The interval that describes the part of the content that has been viewed in minutes."
+        },
+        "xdm:path": {
```

Comment:
Isn't there a pre-existing media grammar that could be used?  Do we really need to invent a new one??

Reply:
This has the same meaning as the "xdm:path" field in schemas/assets/image.schema.json and schemas/assets/video.schema.json. We used the same name for consistency.